ldap3/protocol/rfc4512.py

```diff
@@ -446,7 +446,7 @@ def from_definition(cls, definitions):
                 else:
                     raise LDAPSchemaError('unknown schema definition class')
 
-                splitted = re.split('( NAME | DESC | OBSOLETE| X-| E-' + pattern + ')', object_definition[1:-1])
+                splitted = re.split('( NAME | DESC | OBSOLETE| X-| E-' + pattern + ')', str(object_definition[1:-1]))
```

Comment:
Internal Server Error: /autentica/valida-usuario/
Traceback (most recent call last):
  File "/home/alexandre.odoni/personal/python/env-chamados-cmc/lib/python3.5/site-packages/django/core/handlers/exception.py", line 42, in inner
    response = get_response(request)
  File "/home/alexandre.odoni/personal/python/env-chamados-cmc/lib/python3.5/site-packages/django/core/handlers/base.py", line 187, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/home/alexandre.odoni/personal/python/env-chamados-cmc/lib/python3.5/site-packages/django/core/handlers/base.py", line 185, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/usr/lib/python3.5/contextlib.py", line 30, in inner
    return func(*args, **kwds)
  File "/home/alexandre.odoni/personal/python/projetos/chamados-cmc/chamados-cmc/autentica/views.py", line 32, in valida_usuario
    user = authenticate(username=usuario, password=senha)
  File "/home/alexandre.odoni/personal/python/env-chamados-cmc/lib/python3.5/site-packages/django/contrib/auth/__init__.py", line 74, in authenticate
    user = backend.authenticate(**credentials)
  File "/home/alexandre.odoni/personal/python/env-chamados-cmc/lib/python3.5/site-packages/django_python3_ldap/auth.py", line 23, in authenticate
    return ldap.authenticate(*args, **kwargs)
  File "/home/alexandre.odoni/personal/python/env-chamados-cmc/lib/python3.5/site-packages/django_python3_ldap/ldap.py", line 166, in authenticate
    with connection(password=password, **kwargs) as c:
  File "/usr/lib/python3.5/contextlib.py", line 59, in __enter__
    return next(self.gen)
  File "/home/alexandre.odoni/personal/python/env-chamados-cmc/lib/python3.5/site-packages/django_python3_ldap/ldap.py", line 145, in connection
    raise_exceptions=True,
  File "/home/alexandre.odoni/personal/python/env-chamados-cmc/lib/python3.5/site-packages/ldap3/core/connection.py", line 325, in __init__
    self.bind(read_server_info=True)
  File "/home/alexandre.odoni/personal/python/env-chamados-cmc/lib/python3.5/site-packages/ldap3/core/connection.py", line 590, in bind
    self.refresh_server_info()
  File "/home/alexandre.odoni/personal/python/env-chamados-cmc/lib/python3.5/site-packages/ldap3/core/connection.py", line 1299, in refresh_server_info
    self.server.get_info_from_server(self)
  File "/home/alexandre.odoni/personal/python/env-chamados-cmc/lib/python3.5/site-packages/ldap3/core/server.py", line 436, in get_info_from_server
    self._get_schema_info(connection)
  File "/home/alexandre.odoni/personal/python/env-chamados-cmc/lib/python3.5/site-packages/ldap3/core/server.py", line 414, in _get_schema_info
    self._schema_info = SchemaInfo(schema_entry, connection.response[0]['attributes'], connection.response[0]['raw_attributes']) if result else None
  File "/home/alexandre.odoni/personal/python/env-chamados-cmc/lib/python3.5/site-packages/ldap3/protocol/rfc4512.py", line 287, in __init__
    self.attribute_types = AttributeTypeInfo.from_definition(attributes.pop('attributeTypes', []))
  File "/home/alexandre.odoni/personal/python/env-chamados-cmc/lib/python3.5/site-packages/ldap3/protocol/rfc4512.py", line 439, in from_definition
    splitted = re.split('( NAME | DESC | OBSOLETE| X-| E-' + pattern + ')', object_definition[1:-1])
  File "/home/alexandre.odoni/personal/python/env-chamados-cmc/lib/python3.5/re.py", line 203, in split
    return _compile(pattern, flags).split(string, maxsplit)
TypeError: cannot use a string pattern on a bytes-like object
"POST /autentica/valida-usuario/ HTTP/1.1" 500 191077